app/uk/gov/hmrc/fhregistrationfrontend/views/businessPartners/v2/business_partner_enter_address.scala.html

```diff
@@ -0,0 +1,166 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.fhregistrationfrontend.forms.models.Address
+@import uk.gov.hmrc.fhregistrationfrontend.views.helpers._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers._
+@import uk.gov.hmrc.fhregistrationfrontend.forms.definitions.BusinessPartnersEnterAddressForm._
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(viewHelpers: ViewHelpers,
+layout: layout,
+formHelper: FormWithCSRF,
+textInput: components.text_field
+)
+
+@(form: Form[Address],postAction: Call, partnerName: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+
+@AddressContent = @{AddressUK(
+form,
+AddressUKParams(
+name = s"form.$enterAddressKey",
+label = Messages("fh.business.partners.enterAddress.title","company"),
+hasErrors = form(s"form.postcode").hasErrors,
+error = Some(Messages("fh.business_partners.enter.partner_address.error"))
+),
+viewHelpers = viewHelpers
+)}
+
+
+@fieldName = @{s"form.$enterAddressKey"+"-title"}
+
+
+@layout(title = messages("fh.business.partners.enterAddress.title"), backUrl = Some("#")){
+
+
+@viewHelpers.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.BusinessPartnersChooseAddressController.next, 'novalidate -> "novalidate"){
```

Comment:
we do not need to wrap the error summary in a form - this form wrapper is not required as it makes 2 forms on the page instead of one - you already have a form wrapper around the form starting at L64